python/tank/bootstrap/manager.py

```diff
@@ -59,6 +59,8 @@ def __init__(self, sg_user=None):
                         authentication, simply construct an explicit user object
                         and pass it in.
         :type sg_user: :class:`~sgtk.authentication.ShotgunUser`
+        :param bool allow_config_overrides: Whether to allow the config override
+            environment variable to affect the results of configuration resolutions.
```

Comment:
Also, will this show in the `__repr__`? It forms a fundamental part of the state of an instance of this class and would be great if the repr captured all aspects of the state.